Tools/TestWebKitAPI/Tests/WebKitCocoa/SessionStorage.mm

```diff
@@ -147,3 +148,187 @@ static void runSessionStorageInNewWindowTest(NSString* createWindowJS, ShouldSes
     }];
     TestWebKitAPI::Util::run(&readyToContinue);
 }
+
+namespace TestWebKitAPI {
+
+static void synchronouslyLoadUrl(RetainPtr<TestWKWebView>& webView, RetainPtr<TestNavigationDelegate>& delegate, NSString* url)
```

Comment:
TestWKWebView has synchronouslyLoadRequest

Reply:
Unfortunately, the test hangs if I use any of the synchronouslyLoad* functions more than once. I didn't immediately see what was causing it, so I created a new wrapper around `loadRequest`. I can dig into this further if you'd prefer that.

Reply:
This isn't a blocker for landing this PR.  Look into this later.